andela/bp-esa-frontend
# #164590294 Version 2 search filters

#### What does this PR do?
Implements the v2 filters.

#### Description of Task to be completed?
- Remove the previous UI and its tests.
- Add the new UI according to the mockups.
- Add tests for the new UI.

#### How should this be manually tested?
1. Start the project and log in with your Andela account.
2. Filter for the desired automation reports using the search filter dropdown launched by clicking the button appended to the search input and applying them.
3. The filters should return exactly what you have filtered for.

#### Any background context you want to provide?
- The filter checkboxes are mutually exclusive, i.e. for a specific filter, they can't both be activated.
- `react-date-picker` is set to match version `1.8.0` exactly. This is because some styles are modified to fit the product's aesthetics. Future versions may change the modified class names and we can't risk that.

#### What are the relevant pivotal tracker stories?
* [#164590294](https://www.pivotaltracker.com/story/show/164590294)

#### Screenshots (If applicable)
![image](https://user-images.githubusercontent.com/41262970/54748106-c668ea00-4be1-11e9-92dc-d3cd100f7d3e.png)
